src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -664,11 +664,11 @@
 (rf/reg-event-fx
  :wallet/handle-suggested-routes
  (fn [{:keys [db]} [data]]
-   (let [{:keys [send swap]} (-> db :wallet :ui)
-         skip-processing-routes?  (:skip-processing-suggested-routes? send)
-         clean-user-tx-settings?  (get-in db
-                                          [:wallet :ui :send :custom-tx-settings
-                                           :delete-on-routes-update?])]
+   (let [{:keys [send swap]}     (-> db :wallet :ui)
+         skip-processing-routes? (:skip-processing-suggested-routes? send)
+         clean-user-tx-settings? (get-in db
+                                         [:wallet :ui :send :custom-tx-settings
+                                          :delete-on-routes-update?])]
```

Comment:
this seems specific to the send flow, but not the swap flow. Should be moved into `:wallet/suggested-routes-success` I guess